app/src/test/java/org/mozilla/tv/firefox/ScreenControllerStateMachineTest.kt

```diff
@@ -12,53 +12,51 @@ import org.mozilla.tv.firefox.ScreenControllerStateMachine.Transition.REMOVE_OVE
 
 class ScreenControllerStateMachineTest {
 
-    var currentUrlIsHome = false
-    lateinit var stateMachine: ScreenControllerStateMachine
+    private var currentUrlIsHome = false
```

Comment:
Nit: since this doesn't have to be passed in at init anymore, I think this can be a local variable.